src/main/java/frc/robot/commands/pretest/MotorTestCommand.java

```diff
@@ -23,10 +24,9 @@ public MotorTestCommand(BaseSwerveSubsystem swerveSubsystem, TiltedElevatorSubsy
         addRequirements(swerveSubsystem, tiltedElevatorSubsystem, rollerSubsystem);
 
         addCommands(
-            // lock swerve
-            new InstantCommand(() -> {
-                swerveSubsystem.applyLock();
-            }, swerveSubsystem),
+            new PrintCommand("hello 1"),
```

Comment:
Not the debug prints from when the test sequence was scheduled but not run

Reply:
was that ever fixed, by the way?

Reply:
No, but we can probably continue debugging on main.